.github/workflows/build-and-publish-rule-book.yaml

```diff
@@ -108,10 +108,16 @@ jobs:
         uses: xu-cheng/latex-action@v3
         with:
           extra_system_packages: "inkscape"
+          pre_compile: echo "icu_locale \"${{ matrix.language }}\"" >> index_style.ist && upmendex -s index_style main_${{ matrix.language }} && sed -i 's@\(\\noindent\\textbf{\)\(.\)@\\noindent\\textbf{\2}@g' main_${{ matrix.language }}.ind
           latexmk_shell_escape: true
-          post_compile: "makeindex main_${{ matrix.language }} -s index_style.ist && pdflatex --shell-escape main_${{ matrix.language }}.tex"
           root_file: main_${{ matrix.language }}.tex
 
+      - name: Move file
```

Comment:
two different Move files?

Reply:
Yes, because the deployment steps will deploy from a directory. You move both files to the same directory

Reply:
Argh, no copy paste error xD